config/default/editor.editor.rich_text_wizard.yml

```diff
@@ -38,7 +38,11 @@ settings:
       reversed: true
       startIndex: true
     ckeditor5_sourceEditing:
-      allowed_tags: {  }
+      allowed_tags:
+        - '<p class>'
+        - '<span class>'
+        - '<h1>'
+        - '<div class>'
```

Comment:
@mattdanger This will only affect the wizard's HTML editors, right? Seems OK to me.

Reply:
Correct, this text format is only used by the Wizard. So this should be fine 👍